modules/video/src/lkpyramid.cpp

```diff
@@ -616,11 +640,11 @@ void cv::detail::LKTrackerInvoker::operator()(const Range& range) const
                 }
             }
 
-#if CV_SSE2
-            float CV_DECL_ALIGNED(16) bbuf[4];
-            _mm_store_ps(bbuf, _mm_add_ps(qb0, qb1));
-            ib1 += bbuf[0] + bbuf[2];
-            ib2 += bbuf[1] + bbuf[3];
+#if CV_SIMD128 && !CV_NEON
+            v_float32x4 qf0, qf1;
+            v_recombine(v_interleave_pairs(qb0 + qb1), v_setzero_f32(), qf0, qf1);
```

Comment:
IMO it would be better to use `v_recombine(v_interleave_pairs(qb0), v_interleave_pairs(qb1), qf0, qf1);`. However I don't think it will essentially affect the performance.

Reply:
Addition will be performed as a part of following `v_reduce_sum`